### DISCUSSION TITLE
Small color differences not picked up by Percy

### DISCUSSION THREAD

**Discussion by thomasmattheussen** [percy/cli#884] [Support] [answered]
I've just updated all grey colors in my project and expected Percy to flag these changes in the diff. But it showed no changes from the baseline. The colors aren't really all that different, so my best guess is Percy doesn't consider this as "enough" change from the baseline? Maybe I missed it, but I can't seem to find anything in the docs about this either.

(these are the colors that changed)

"#F3F4F4" => "#F2F2F2"
"#EDEDED" => "#E6E6E6"
"#D7D7D7" => "#CCCCCC"
"#B8B8B8" => "#B3B3B3"
"#919191" => "#999999"
"#696969" => "#808080"
"#474747" => "#666666"
"#2F2F2F" => "#4D4D4D"
"#212121" => "#333333"
"#1C1C1C" => "#1A1A1A"

Is my guess correct? In that case I think this should be documented somewhere.
Or is there something else going on?

**Comment by cadeParade** [comment]
Hi @thomasmattheussen , thanks for this report. Do you have a link to a build I can check out?

**Reply by thomasmattheussen** [reply]
https://percy.io/78ddb253/chameleon-hub/builds/17364420/unchanged/974559653?browser=chrome&browser_ids=23&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1280&widths=1280

**Comment by cadeParade** [comment, marked as answer]
Thanks for the link!
I've checked this snapshot with different settings, and you're right, Percy did not highlight the greys changing because the colors are very close together. 
On the projects settings page you can adjust how sensitive your builds are to color changes in the "diff-sensitivity" section. 
<img width="1148" alt="CleanShot 2022-04-14 at 14 37 09@2x" src="https://user-images.githubusercontent.com/3179218/163480927-de8eb046-165e-41b6-b330-cba333eaaa71.png">

The snapshot you link has no diff in the default sensitivity or the one below, but the diff does show up with the most strict diff sensitivity.

You are correct that it is not documented anywhere, however @j000shDotCom is working on that currently,  so hopefully it will be clearer to customers in the future.

**Reply by thomasmattheussen** [reply]
Ok, thanks!

**Reply by cadeParade** [reply]
For closure, here is where the documentation now lives: https://docs.percy.io/docs/diff-sensitivity
